Description: Unambiguous desktop entry name and definition
 The patch is needed to distinguish this KCM from the base one provided
 by Baloo. The Name field has been changed in baloo-kcmadv, but its
 translations have not; both Description and its translations have not
 been changed. So this patch fixes Descriptions and remove
 all translations for now.
Forwarded: no
Author: Luigi Toscano <luigi.toscano@tiscali.it>
Last-Update: 2014-10-05

--- baloo-kcmadv-0~git20140427.orig/kcm_baloofileadv.desktop
+++ baloo-kcmadv-0~git20140427/kcm_baloofileadv.desktop
@@ -11,77 +11,7 @@ X-DocPath=kcontrol/kcm_baloofileadv/inde
 X-KDE-System-Settings-Parent-Category=workspace-appearance-and-behavior
 
 Name=Desktop Search Advanced
-Name[bs]=Pretraga površi
-Name[ca]=Cerca a l'escriptori
-Name[ca@valencia]=Cerca a l'escriptori
-Name[cs]=Vyhledávací služby
-Name[da]=Skrivebordssøgning
-Name[de]=Desktop-Suche
-Name[el]=Αναζήτηση στην επιφάνεια εργασίας
-Name[es]=Búsqueda de escritorio
-Name[fi]=Työpöytähaku
-Name[fr]=Recherche sur le bureau
-Name[gl]=Busca no escritorio
-Name[hu]=Asztali keresés
-Name[ia]=Cerca de Scriptorio
-Name[it]=Ricerca dal desktop
-Name[ko]=데스크톱 검색
-Name[lt]=Darbastalio paieška
-Name[nb]=Skrivebordssøk
-Name[nds]=Schriefdischsöök
-Name[nl]=Bureaublad-zoekopdracht
-Name[pa]=ਡੈਸਕਟਾਪ ਖੋਜ
-Name[pl]=Wyszukiwanie na pulpicie
-Name[pt]=Pesquisa no Ambiente de Trabalho
-Name[pt_BR]=Pesquisa no desktop
-Name[ru]=Поиск по меткам и содержимому
-Name[sk]=Hľadanie na ploche
-Name[sl]=Namizno iskanje
-Name[sr]=Претрага површи
-Name[sr@ijekavian]=Претрага површи
-Name[sr@ijekavianlatin]=Pretraga površi
-Name[sr@latin]=Pretraga površi
-Name[sv]=Skrivbordssökning
-Name[tr]=Masaüstü Araması
-Name[uk]=Стільничний пошук
-Name[x-test]=xxDesktop Searchxx
-Name[zh_CN]=桌面搜索
-Name[zh_TW]=桌面搜尋
-Comment=Configure Desktop Search
-Comment[ca]=Configuració de la cerca a l'escriptori
-Comment[ca@valencia]=Configuració de la cerca a l'escriptori
-Comment[cs]=Nastavit vyhledávací služby
-Comment[da]=Indstil skrivebordssøgning
-Comment[de]=Desktop-Suche einrichten
-Comment[el]=Διαμόρφωση αναζήτησης επιφάνειας εργασίας
-Comment[es]=Configurar la búsqueda de escritorio
-Comment[fi]=Työpöytähaun asetukset
-Comment[fr]=Configurer la recherche sur le bureau
-Comment[gl]=Configurar a busca de escritorio
-Comment[hu]=Asztali keresés beállítása
-Comment[ia]=Configura cerca de scriptorio
-Comment[it]=Configura ricerca dal desktop
-Comment[ko]=데스크톱 검색 설정
-Comment[lt]=Konfigūruoti darbastalio paiešką
-Comment[nb]=Sett opp skrivebordssøk
-Comment[nds]=Schriefdischsöök instellen
-Comment[nl]=Bureaubladzoeken instellen... 
-Comment[pl]=Konfiguruj znajdywanie na pulpicie
-Comment[pt]=Configurar a Pesquisa no Ambiente de Trabalho
-Comment[pt_BR]=Configurar a pesquisa no desktop
-Comment[ru]=Настройка службы поиска
-Comment[sk]=Nastaviť hľadanie na ploche
-Comment[sl]=Nastavi Namizno iskanje
-Comment[sr]=Подешавање претраге површи
-Comment[sr@ijekavian]=Подешавање претраге површи
-Comment[sr@ijekavianlatin]=Podešavanje pretrage površi
-Comment[sr@latin]=Podešavanje pretrage površi
-Comment[sv]=Anpassa skrivbordssökning
-Comment[tr]=Masaüstü Aramasını Yapılandır
-Comment[uk]=Налаштовування стільничного пошуку
-Comment[x-test]=xxConfigure Desktop Searchxx
-Comment[zh_CN]=配置桌面搜索
-Comment[zh_TW]=設定桌面搜尋
+Comment=Advanced Configuration for Desktop Search
 X-KDE-Keywords=Search, File, Baloo
 X-KDE-Keywords[ca]=Cerca, Fitxer, Baloo
 X-KDE-Keywords[ca@valencia]=Cerca, Fitxer, Baloo
